src/lib/controllers/ui-controller.js

```diff
@@ -870,25 +958,29 @@ export default class UIController {
   }
 
   updateWordUsageExamples (wordUsageExamplesData) {
-    this.addMessage(this.api.l10n.getMsg('TEXT_NOTICE_WORDUSAGE_READY'))
+    this.store.commit('ui/addMessage', this.api.l10n.getMsg('TEXT_NOTICE_WORDUSAGE_READY'))
     this.store.commit('app/setWordUsageExamplesData', wordUsageExamplesData)
   }
 
   clear () {
+    this.store.commit(`app/resetStatusData`)
+    this.store.commit(`app/resetDefDataReady`)
     this.store.commit(`app/resetInflData`)
     this.store.commit(`app/resetTreebankData`)
-    if (this.hasUiModule('panel')) { this.getUiModule('panel').vi.clearContent() }
-    if (this.hasUiModule('popup')) { this.getUiModule('popup').vi.clearContent() }
```

Comment:
where is component content clearing happening now?  panelCompact still has a clearContent method but I don't think it's called.

Reply:
UI components do not have their own data related to business logic now. All such data is now either in the global Vuex store or in the UI controller's API objects that are shared with the UI components. So there is no reason to clear component's data. Once the data is cleared within the UI controller, components will see this change and will redraw itself.

UI components can have its own local state (like expanded/collapsed variables), but that usually won't require any special cleaning. But if it does, components watch changes of Vuex store data and update its own state once certain pieces of Vuex data change.

`clearContent()` is a piece of legacy code that I've missed to remove. Deleted it now.

Reply:
ok, thank you!